public/api/docs/v3/swagger.yaml

```
@@ -7,9 +7,86 @@ externalDocs:
   description: Find out more about Swagger
   url: https://swagger.io/
 paths:
+  "/api/v3/schools":
+    get:
+      summary: Retrieve multiple ECF schools scoped to cohort
```

```suggestion
      summary: Retrieve multiple schools scoped to cohort
```

'ECF' is outdated terminology we should avoid. @mark-avery-dfe